app/main.py

```diff
@@ -2,5 +2,17 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_results = {}
+
+    def wrapper(*args, **kwargs) -> None:
+        arg_key = (args, frozenset(kwargs.items()))
+        if arg_key in cache_results:
+            print("Getting from cache")
+            return cache_results[arg_key]
```

Comment:
You can use a single return statement out of the if/else statement instead of two inside it.
try to rewrite your decorator not to have `return` duplicated